scrapy_crawlera/middleware.py

```diff
@@ -30,6 +30,7 @@ class CrawleraMiddleware(object):
     force_enable_on_http_codes = []
     max_auth_retry_times = 10
     enabled_for_domain = {}
+    apikey = ""
```

Comment:
Is this change necessary or related to the issue?

Reply:
It is not necessary, just added for the pylint doesn't show a warning of `Instance of 'CrawleraMiddleware' has no 'apikey' member pylint(no-member)` and be explicit that starts as an empty string as default